doc/subsystems/bluetooth/sm-pics.rst

```diff
@@ -69,6 +69,7 @@ TSPC_SM_5_1	True		Encryption Setup using STK (C.3)
 TSPC_SM_5_2	True		Encryption Setup using LTK (O)
 TSPC_SM_5_3	True		Slave Initiated Security (C.1)
 TSPC_SM_5_4	True		Slave Initiated Security - Master response(C.2)
+TSPC_SM_5_5	False		Link Key Conversion Function h7 (C.4)
```

Comment:
We do have h7 implemented (take a look at smp_h7() in smp.c).

Reply:
OK